src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -1,20 +1,33 @@
 package blackjack.domain
 
-data class Players(private val players: List<Player> = listOf()) : List<Player> by players {
+import blackjack.domain.card.CardDeck
+import blackjack.domain.strategy.draw.DrawStrategy
 
-    fun eachAcceptCards(cardDeck: CardDeck) {
-        players.forEach {
-            it.hit(cardDeck.next())
+class Players private constructor(players: List<Player>) {
+
+    private val _players = players.map { it.copy() }
+    val players: List<Player>
+        get() = _players
+            .map {
+                it.copy()
+            }
```

Comment:
```suggestion
   val players = players.map { it.copy() }
```
private 변수가 mutable하지 않기 때문에 이렇게 사용해도 될 것 같아요